pkg/server/app/app.go

```diff
@@ -96,8 +97,14 @@ func (ops *AppOperations) Create(user *storage.User, app *App) error {
 		return teresa_errors.NewInternalServerError(err)
 	}
 
+	defer func() {
+		if Err != nil {
```

Comment:
You're right, but you know, as we say in Portuguese "o seguro morreu de velho". Could you add a test to prevent this fearful behavior ?